draft-ietf-wimse-arch.md

```diff
@@ -79,8 +79,20 @@ Attestation is the function through which a task verifies the identity of a sepa
 
 ## Workload Identity
 
+A workload identity may be composed of many attributes describing different aspects of a workload. These attributes may describe different sets of information such as a workload's role within a system, the software that the workload runs or the hardware that the workload runs on. Sets of attributes may be determined by different authorities running in different parts of the system. WIMSE defines a **Workload Identifier** to used in representing and referencing the workload.  It allows different authorities of sets identity attributes to reference the workload in a consistent way.
+
+The Workload Identifier consists of a concise string allocated within a namespace defined by a Trust Domain. This Workload Identifier is present in Workload Identity Tokens and X.509 certificates issued by the authority for the Trust Domain. The Workload Identifier is used to associate additional identity attributes to the workload through the use of tokens (workload attribute tokens?) or online look up services. It may also be used directly in authorization calculations and audit logs.
+
+The Trust Domain consists of a string that matches the format of a fully qualified domain name. It is the intent that a Trust Domain is actually a domain name registered to the organization defining the Trust Domain, but this may not be true in all cases. The Trust Domain also maps to the issuer of cryptographically signed Workload Identity Tokens (WIT) or X.509 Certificate. The association between a Trust Domain and the cryptographic root of the signing authority for that Trust Domain must be made securely through an out-of-band mechanisms. [TODO: where should mechanisms be defined?]
```

Comment:
I left it open here, but I really think the trust domain should be represented within the identifier in a consistent way, however alternatively it could be a separate identifier for a trust domain that mist be represented in a certificate or token field.